cogeo_mosaic/backends/base.py

```diff
@@ -16,6 +16,8 @@ class BaseBackend(AbstractContextManager):
 
     path: str
     mosaic_def: MosaicJSON
+    _backend_name: str
+    _file_byte_size: Optional[int]
```

Comment:
Added more info about the backend and possibility to store the file size (for the CLI info)